draft/jumpstarter.rst

```diff
@@ -64,6 +64,115 @@ the other features of Python's ``async/await`` programming model.
 Specification
 =============
 
+Jumpstarter is a Python implementation of an `Actor System`_ (which utilizies the `Actor Model`_). There
+are three fundamental axioms within the actor model (quoting the previous Wikipedia link): 
+
+(Start Blockquote)
```

Comment:
See [RST spec](https://docutils.sourceforge.io/docs/user/rst/quickref.html#block-quotes) for blockqoutes.